src/fees/saga.ts

```diff
@@ -162,11 +187,7 @@ export function* estimateInviteFee(tokenAddress: string) {
 
   // We must add a static amount here for the transfer because we can't estimate it without sending the approve tx first.
   // If the approve tx hasn't gone through yet estimation fails because of a lack of allowance.
-  const feeInfo: FeeInfo = yield call(
-    calculateFeeForTx,
-    tx.txo,
-    new BigNumber(STATIC_ESCROW_TRANSFER_GAS_ESTIMATE)
-  )
+  const feeInfo: FeeInfo = yield call(calculateFeeForTx, tx.txo)
```

Comment:
Hmm, probably you already thought about this, but it seems that we are calculating a different fee for `estimateInviteFee`. I'm wondering if this could break something... 🤔

Reply:
lol yeah i totally broke this but this is fee estimation for sending to escrow which is no longer supported...i thought about making `calculateFeeForTx` accept a static fee for addition AND a fee multiplier, but figured it's not worth the ugly function signature and opted to break this unused flow instead. i have a ticket in the backlog to remove escrow from the wallet